modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/reagent.dm

```diff
@@ -0,0 +1,10 @@
+
+/datum/reagent/overdose_process(mob/living/M, delta_time, times_fired)
+	// Because these chemicals shouldn't bear the same weight as normal / debatably more harmful chemicals.
+	if(name == "dopamine")///This one also shouldn't have any negative mood effect.
+		return
+	if(name == "succubus milk" || name == "incubus draft" || name == "Camphor" || name == "Pentacamphor")
+		to_chat(M, span_userdanger("You feel like you took too much [name]!"))
+		M.add_mood_event("[type]_overdose", /datum/mood_event/minor_overdose, name)
+		return
+	. = ..()
```

Comment:
Please, _please_, fix this.

This should be handled by the overdose process of those reagents, rather than being handled here.

Reply:
Wait, I didn't even look at this when I was copypasting, what in the actual fuck?

Reply:
Also, imagine type checking.